lib/manager/gradle/shallow/extract.spec.ts

```diff
@@ -136,4 +137,73 @@ describe('manager/gradle/shallow/extract', () => {
       },
     ]);
   });
+
+  it('works with dependency catalogs', async () => {
+    const fsMock = {
+      'gradle/libs.versions.toml': dedent`
+        [versions]
+        detekt = "1.17.0"
+        kotest = "4.6.0"
+        publish-on-central = "0.5.0"
+
+        [libraries]
+        detekt-formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }
+        kotest-assertions-core-jvm = { module = "io.kotest:kotest-assertions-core-jvm", version.ref = "kotest" }
+        kotest-runner-junit5 = { module = "io.kotest:kotest-runner-junit5", version.ref = "kotest" }
+        mockito = { group = "org.mockito", name = "mockito-core", version = "3.10.0" }
+
+        [bundles]
+        kotest = [ "kotest-runner-junit5", "kotest-assertions-core-jvm" ]
+
+        [plugins]
+        detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
```

Comment:
We should add one without `version.ref`, i.e:

```toml
detekt = { id = "io.gitlab.arturbosch.detekt", version = "1.18.0" }
```

Reply:
I'm adding the TOML you produced in the example project as an additional test